source/routes/create/bullet.js

```diff
@@ -0,0 +1,58 @@
+const PouchDB = require('pouchdb');
+PouchDB.plugin(require("pouchdb-find"));
+const db = new PouchDB("db");
+
+// Send in json form of the bullet and it gets sent to the database
+module.exports = {
+    '/create/bullet': {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-sp21-group13/cse110-sp21-group13/pullRequest?prid=7402063)
```suggestion
    "/create/bullet": {
```